### DISCUSSION TITLE
What to do when my transaction count is well over 2 million

### DISCUSSION THREAD

**Discussion by sigaloid** [TriliumNext/Trilium#2322] [General]
![image](https://user-images.githubusercontent.com/69441971/141056777-8c999f24-637c-40db-9517-75a5ca43aa8c.png)

I accidentally set my note revision timeout to 1 second, not 1 minute, and didn't realize for a few days... 

Can I export `root` and reimport it? What are the implications of that?

Can I somehow delete all revisions of all notes? I don't mind losing them if syncs don't take so long...

**Comment by zadam** [comment]
If you don't mind losing history then it's probably the easiest to make an export and then import into an empty document. By doing that you lose the history, but otherwise the whole content, attributes etc. will be kept. Settings in the options will be also resetted.

**Reply by sigaloid** [reply]
Thanks :D
